File: source/back-end/src/main/java/be/pxl/backend/service/ICommentService.java
package be.pxl.backend.service;

import java.util.List;
import be.pxl.backend.entity.Comment;

public interface ICommentService {

	public Comment find(int id);
	
	public List<Comment> all();
	
	public void persist(Comment comment);
	
	public void delete(int id);
	
	public void update(int id, Comment comment);
	
}

File: Source/frontend/frontend.Service/ICommentService.cs
using frontend.Domain;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace frontend.Service
{
    public interface ICommentService
    {
        List<Comment> All();
        Comment Find(int id);
        void Add(Comment comment);
        void Update(Comment comment);
        void Delete(int id);
    }
}
